src/renderer/routes/test_sequencer_panel/components/TestSequencerInfo.tsx

```diff
@@ -51,15 +57,20 @@ const TestSequencerView = () => {
     tSSendJsonMessage(testSequenceStopRequest(tree));
     setIsLocked(false);
   };
-  const testSetSave = useTestSetSave();
-  const testSetImport = useTestSetImport();
+  const projectImport = useImportProject();
+  const saveProject = useSaveProject();
+  const closeProject = useCloseProject();
   const [isImportModalOpen, setIsImportModalOpen] = useState(false);
   const handleClickImportTest = () => {
     setIsImportModalOpen(true);
   };
 
   return (
     <LockedContextProvider>
+      <TestSequencerProjectModal
```

Comment:
we should probably make a component <Modals/> and have all the modals inside of it just to make the code ultra clean. We should also have the states of the modals in a zustand store useModalsState and specify the methods to opening and closing a modal, rather than have local fields. 

You don't need to do this, but in the future it would be a good practice to start doing so because it can get real messy :) this is just me putting my thoughts on the PR.